totalRP3/Core/UITools.lua

```diff
@@ -579,7 +579,13 @@ local function getCompanionOwner(unitType, targetType)
 	local ownerName;
 	local ownerRealm;
 
-	if C_TooltipInfo then
+	if UnitOwnerGUID then
+		local ownerGUID = UnitOwnerGUID(unitType);
+
+		if ownerGUID ~= nil then
+			ownerName, ownerRealm = select(6, GetPlayerInfoByGUID(ownerGUID));
```

Comment:
Actually, if I wanted to nitpick, that part is present in both ifs, we could get it out of there

Reply:
We could, but that other branch'll be deleted at some point anyway.

Reply:
Fair